posts/Fabio/2018-02-13-hibernate-ogm-5-3-CR1-released.adoc

```diff
@@ -0,0 +1,81 @@
+= "Hibernate OGM 5.3 CR1 is out"
+Fabio Massimo Ercoli
+:awestruct-tags: [ "Hibernate OGM", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+We're happy to announce the release of:
+http://hibernate.org/ogm/releases/5.3/#get-it[Hibernate OGM 5.3 CR1]
+
+== Components upgrade
+
+* Hibernate ORM version is upgraded to 5.2
+* Hibernate Search version is upgraded to 5.9
+
+== Infinispan embedded sequences and id generation improvements
+
+The Infinispan Clustered Counters, used for sequence generation, are now created at startup.
+The dialect no longer needs to check if the counter already exists for every use.
```

Comment:
would change this to something like:

When using @SequenceGenerator, Hibernate OGM will create Clustered Counter at start up. This change will lead to better performance because it doesn't have to check if the Clustered Counter already exists each time.